.github/workflows/ci.yml

```diff
@@ -1169,7 +1170,9 @@ jobs:
           secret_vault_secretId: ${{ secrets.VAULT_SECRET_ID }}
           secret_vault_address: ${{ secrets.VAULT_ADDR }}
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
-      - run: exit ${{ ((contains(needs.*.result, 'cancelled') || contains(needs.*.result, 'failure')) && 1) || 0 }}
+      - run: |
+          EXIT_CODE="${{ ((contains(needs.*.result, 'cancelled') || contains(needs.*.result, 'failure')) && 1) || 0 }}"
+          exit "$EXIT_CODE"
```

Comment:
The expression flagged here by `shellcheck` is actually a GitHub Actions expression that will be templated before the Bash. Should be fine but `shellcheck` probably mis-detects this ...